class CrashCreateError(Exception):
    pass


class CrashGetError(Exception):
    pass
